themes/hugo_theme_robust/layouts/partials/header_after.html

```diff
@@ -58,20 +58,20 @@
 					  <li><a href="http://owi.usgs.gov/R/training.html">Training Schedule</a></li>
 					
 					  <li>
-						  <a href="http://dev-owi.usgs.gov-website.s3-website-us-west-2.amazonaws.com/R/intro-curriculum/">
+						  <a href="http://owi.usgs.gov/R/training-curriculum/intro-curriculum">
```

Comment:
I wonder if you could put in: {{ .Site.BaseURL }}
that way dev and prod would work.